### DISCUSSION TITLE
Why do the links in the navbar keep increasing every time I switch the docs?

### DISCUSSION THREAD

**Discussion by kakarj** [fuma-nama/fumadocs#1078] [Q&A] [answered]
https://github.com/user-attachments/assets/fb97b2a1-43e6-407f-b8ba-9d97b3d54f76

both the docs have same links: 

<img width="733" alt="Screenshot 2024-11-20 at 8 32 17 AM" src="https://github.com/user-attachments/assets/b392a540-acab-42c3-9dc4-67a6c1698d69">
<img width="813" alt="Screenshot 2024-11-20 at 8 32 08 AM" src="https://github.com/user-attachments/assets/6649a670-0bf3-4443-8aa3-0b40ebe7d18d">

**Comment by fuma-nama** [comment]
Hmm mind to provide a reproduction?

**Reply by kakarj** [reply]
<img width="263" alt="Screenshot 2024-11-20 at 3 43 56 PM" src="https://github.com/user-attachments/assets/4092489f-4900-4df0-b66d-8d79217c8256">

Currently, each module contains only a single simple text:

<img width="485" alt="Screenshot 2024-11-20 at 3 46 48 PM" src="https://github.com/user-attachments/assets/0ae500a3-b16e-405d-977a-e9f81f383edb">

**Reply by fuma-nama** [reply]
You could provide a GitHub repo or CodeSandbox that reproduces the problem, I cannot reproduce the issue with a similar setup.

**Reply by kakarj** [reply]
https://github.com/kakarj/odl/tree/main/content/docs

**Reply by fuma-nama** [reply]
I think the error already reminded of this, you cannot have multiple items with the same URL, as we use it as a unique key

![image](https://github.com/user-attachments/assets/54375f44-9c70-4664-a129-64da76df9e10)

**Reply by kakarj** [reply]
Is there another way to handle this? I can update the Discord URL, but I can't change the forum or YouTube links.

**Reply by fuma-nama** [reply, marked as answer]
You can simply remove duplicated urls
